src/api/raw.rs

```diff
@@ -265,6 +265,17 @@ impl RawApi {
         }
     }
 
+    /// Ingress constructor
+    pub fn v1beta1Ingress() -> Self {
+        Self {
+            group: "extensions".into(),
+            resource: "ingresses".into(),
+            prefix: "apis".into(),
+            version: "v1beta1".into(),
+            ..Default::default()
```

Comment:
looks correct to me. had a double check in https://docs.rs/k8s-openapi/0.5.1/src/k8s_openapi/v1_15/api/extensions/v1beta1/ingress.rs.html#352-366